gateway/Cargo.toml

```diff
@@ -51,8 +51,10 @@ phf = "0.11"
 
 # actix_auth
 actix-web-httpauth = "0.8"
-openidconnect = { version = "3.4", default-features = false }
-reqwest = { version = "0.11", default-features = false, features = [
+openidconnect = { git = "https://github.com/multun/openidconnect-rs.git", branch = "updated-at-string", features = [
```

Comment:
We should add a comment with the following link: https://github.com/ramosbugs/openidconnect-rs/pull/165